Minimum Working Solution.ipynb

```diff
@@ -14,57 +14,27 @@
-    "file1 = open('doc1.txt', 'r')"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 136,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "file1contents = file1.read()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 156,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "file2 = open('doc2.txt', 'r')\n",
-    "file2contents = file2.read()\n",
-    "\n",
-    "file3 = open('doc3.txt', 'r')\n",
-    "file3contents = file3.read()\n",
-    "\n",
-    "file4 = open('doc4.txt', 'r')\n",
-    "file4contents = file4.read()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 169,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "file5 = open('doc5.txt', errors = 'ignore')\n",
-    "file5contents = file5.read()\n",
-    "#special characters in this file, needed to ignore so that we can read it"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 170,
-   "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "FileNotFoundError",
+     "evalue": "[Errno 2] No such file or directory: 'doc1.txt'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-2-dfa97e508dc3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0mall_files_contents\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfilename\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfilenames\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m     \u001b[0;32mwith\u001b[0m \u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfilename\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0merrors\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'ignore'\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      5\u001b[0m         \u001b[0mall_files_contents\u001b[0m \u001b[0;34m+=\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: 'doc1.txt'"
+     ]
+    }
+   ],
    "source": [
-    "file6 = open('doc6.txt', 'r')\n",
-    "file6contents = file6.read()"
+    "filenames = ['doc1.txt', 'doc2.txt', 'doc3.txt', 'doc4.txt', 'doc5.txt', 'doc6.txt']\n",
+    "all_files_contents = ''\n",
+    "for filename in filenames:\n",
+    "    with open(filename, errors='ignore') as file:\n",
+    "        all_files_contents += file.read()\n"
```

Comment:
This is the code I added so this is what you should look at. It might be easier if you just accept my PR and then you can checkout the code with a git pull and you can look at it in Jupyter. Let me know if you need a Git primer to help you understand what PRs are and all that stuff.

Reply:
After a quick search of Git primers, I think if you have one, it might be useful for me to have a look at!